cirq-google/cirq_google/json_resolver_cache.py

```diff
@@ -33,7 +33,6 @@ def _class_resolver_dictionary() -> Dict[str, ObjectFactory]:
         'GoogleNoiseProperties': cirq_google.GoogleNoiseProperties,
         'SycamoreGate': cirq_google.SycamoreGate,
         'GateTabulation': cirq_google.GateTabulation,
-        'GridDevice': cirq_google.GridDevice,
```

Comment:
[No change needed] As discussed, this is safe because `GridDevice` has not yet appeared in a Cirq release.